### PR TITLE
Bug fix in boundary parsing

### PR DESCRIPTION
**Pull request by na6vj**
This was preventing all records from being read

### CODE REVIEW

File: querymonitor/README.md [hunk 1]
```diff
@@ -10,6 +11,9 @@ The data collected is suitable for offline analysis to obtain:
 * Node response times & volumes
 * system behaviour by time
 * SLO compliance
+* Tablet followe lag analysis
```
Comment (eugeneckim) [inline querymonitor/README.md:14]: Type "followe" should be "follower"

File: querymonitor/README.md [hunk 2]
```diff
@@ -127,10 +131,46 @@ EDT                  systemq  cqlcount  sys_gt120  cql_gt120  sys_dc3  cql_dc3
...
+`slow_queries    `: "Large" query counts that take a "Long time"
+`slow_tables     `: By-table analysis - Latency  by query type 
+`summary_cql     `: By-Date/Time analysis of queries (in 10-minute slices)
+`tables          `: YCQL/YSQL table details 
+`tabletmetric    `: currently `follower_lag` per tablet, timestamped.
+`tablets         `: tablet details 
+`ycql            `: Query details collected (raw) for YCQL
+`ysql            `: Query details collected (raw) for YSQL
+
+You can getthis list (without the description) using the generated sqlite database:
```
Comment (eugeneckim) [inline querymonitor/README.md:164]: "getthis" should be "get this"

**Review by na6vj — COMMENTED** [review]
fixed 2 typos

**Review by eugeneckim — APPROVED** [review]
LGTM

**Pull request merged**